examples/troopjs_require/.gitignore

```diff
@@ -0,0 +1,51 @@
+node_modules/*
+!node_modules/jquery/dist/jquery.js
+!node_modules/mu-emitter/main.js
+!node_modules/mu-emitter/config.js
+!node_modules/mu-emitter/handler.js
+!node_modules/mu-emitter/executor.js
+!node_modules/mu-emitter/error.js
+!node_modules/mu-error/factory.js
+!node_modules/mu-getargs/main.js
+!node_modules/mu-getargs/src/getargs.js
+!node_modules/mu-jquery-destroy/main.js
+!node_modules/mu-jquery-destroy/src/jQueryDestroy.js
+!node_modules/mu-jquery-hashchange/jquery.hashchange.js
+!node_modules/mu-merge/main.js
+!node_modules/mu-selector-set/main.js
+!node_modules/mu-selector-set/src/SelectorSet.js
+!node_modules/mu-selector-set/src/MappedLists.js
+!node_modules/mu-selector-set/src/matchesSelector.js
+!node_modules/mu-selector-set/src/classifier.js
+!node_modules/mu-selector-set/src/Subsets.js
+!node_modules/mu-selector-set/src/splitter.js
```

Comment:
You can probably use a glob pattern here `!node_modules/mu-selector-set/src/*`, unless you use any files in the `src` folder.